docs/source/guide/training-edge.rst

```diff
@@ -75,7 +76,8 @@ e.g. as logits of a categorical distribution.
             return {'score': score}
     
         def forward(self, graph, h):
-            # h contains the node representations computed from the GNN above.
+            # h contains the node representations computed from the GNN defined
+            # in the node classification section.
```

Comment:
"the node classification section" -> "section 5.1 Node Classification/Regression"